NomadUI/Widgets/NUIButton.cpp

```
@@ -36,6 +36,21 @@ void NUIButton::onRender(NUIRenderer& renderer) {
     
     auto bounds = getBounds();
     float radius = theme->getBorderRadius();
+
+    // ANIMATION: "Squish" effect on pressstill 
+    if (pressed_) {
+        float scale = 0.96f;
```

Clang (14.0.6)

[warning] 0.96f is a magic number; consider replacing it with a named constant

(cppcoreguidelines-avoid-magic-numbers,readability-magic-numbers)